pkgs/tools/misc/figurine/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildGoModule
+}:
+
+let
+  version = "1.3.0";
+in
+buildGoModule {
+  pname = "figurine";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "arsham";
+    repo = "figurine";
+    rev = "v${version}";
+    hash = "sha256-1q6Y7oEntd823nWosMcKXi6c3iWsBTxPnSH4tR6+XYs=";
+  };
+
+  vendorSha256 = "sha256-mLdAaYkQH2RHcZft27rDW1AoFCWKiUZhh2F0DpqZELw=";
```

```suggestion
  vendorHash = "sha256-mLdAaYkQH2RHcZft27rDW1AoFCWKiUZhh2F0DpqZELw=";
```

`vendorSha256` is deprecated. Please use `vendorHash` instead (same value).